app/Services/GitHub/IssueCreateService.php

```
@@ -0,0 +1,116 @@
+    }
+
+    /**
+     * Initialize cache for the current repository
+     */
+    private function initializeCache(string $repo): void
+    {
+        if ($this->currentRepo !== $repo) {
+            $this->memoizedData = [];
+            $this->currentRepo = $repo;
+        }
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'initializeCache'. (Unused Code Rules)

(UnusedPrivateMethod)